cpp/cmake/Dependencies.cmake

```diff
@@ -38,15 +38,16 @@ else(DEFINED ENV{RAFT_PATH})
   set(RAFT_DIR ${CMAKE_CURRENT_BINARY_DIR}/raft CACHE STRING "Path to RAFT repo")
 
   ExternalProject_Add(raft
-    GIT_REPOSITORY    https://github.com/rapidsai/raft.git
-    GIT_TAG           b58f97f2b5382a633e43daec31b26adf52e19a3b
+    GIT_REPOSITORY    https://github.com/aschaffer/raft.git
+    GIT_TAG           d513d21474f851db64c68095db1ea47eeeb1c740
     PREFIX            ${RAFT_DIR}
     CONFIGURE_COMMAND ""
     BUILD_COMMAND     ""
     INSTALL_COMMAND   "")
 
   # Redefining RAFT_DIR so it coincides with the one inferred by env variable.
-  set(RAFT_DIR ${RAFT_DIR}/src/raft/ CACHE STRING "Path to RAFT repo")
+  #set(RAFT_DIR ${RAFT_DIR}/src/raft/ CACHE STRING "Path to RAFT repo")
+  set(RAFT_DIR ${RAFT_DIR}/src/raft/)
```

Comment:
any particular reason for not caching this variable?

Reply:
For more, see slack thread: https://nvidia.slack.com/archives/CKTK6TV9T/p1593472169326300?thread_ts=1593468766.323900&cid=CKTK6TV9T

Reply:
We probably want keep one or the other and minimize commented out lines in the build, right? Does it fail even with the `Path to RAFT repo` description?